ui.apps/pom.xml

```diff
@@ -47,15 +48,42 @@
                     <!-- overwrite rep:ACE nodes defining policies for acs-aem-commons-owned service principals -->
                     <accessControlHandling>merge</accessControlHandling>
                     <packageType>application</packageType>
-                    <dependencies>
-                        <dependency>
-                            <group>day/cq60/product</group>
-                            <name>cq-content</name>
-                            <version>[6.5.6,)</version><!-- version shipped with AEM 6.5.0 GA -->
-                        </dependency>
-                        <!-- cannot depend on SP 10 as never satisfied on AEMCloud -->
-                    </dependencies>
                 </configuration>
+                <executions>
+                    <execution>
+                        <id>default-package</id>
```

Comment:
the id for metadata is `default-metadata`. Please separate the configuration per goal/execution id.